index.html

```diff
@@ -19,22 +19,203 @@
+    <section class="my-achievements">
+     <br><br> <h2>My Recent<br> Works</h2>
+      <span class="division"></span>
+      <div class="achievements-container">
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            <img src="./assets/images/numberone.png" alt="Image1">
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="the-button">See project</button>
+          </div>
+        </div>
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            <img src="./assets/images/numberone.png" alt="Image1">
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="the-button">See project</button>
+          </div>
+        </div>
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            <img src="./assets/images/numberone.png" alt="Image1">
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="the-button">See project</button>
+          </div>
+        </div>
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            <img src="./assets/images/numberone.png" alt="Image1">
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="the-button">See project</button>
+          </div>
+        </div>
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            <img src="./assets/images/numberone.png" alt="Image1">
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
```

Comment:
- [optional] great job implementing all the changes. Please kindly make your cards responsive up till the desktop version(768px). This will help allow people to access content using multiple device resolutions.
![Screenshot (1106)_LI](https://github.com/ngalamac/Ngala-Portfolio/assets/65298048/42f08965-f7c3-4600-9503-146f7f1835b9)